.buildkite/pipeline.build.yml

```diff
@@ -486,6 +486,16 @@
       --test_env=CONDA_DEFAULT_ENV
       python/ray/tests/...
 
+- label: ":book: Doctest (CPU)"
+  instance_size: medium
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
+    - ./ci/env/env_info.sh
+    - DOC_TESTING=1 ./ci/env/install-dependencies.sh
+    - bazel test --config=ci $(./scripts/bazel_export_options)
+      --test_tag_filters=doctest,-gpu
+      python/ray/... doc/source/...
```

Comment:
How would I disable retries? I don't think we should retry this test.

Reply:
I think you can specify the flag again to override it